pyproject.toml

```
@@ -89,15 +89,18 @@ doc = [
     "pypandoc==1.11",
     "pytest-sphinx==0.5.0",
     "pythreejs==2.4.2",
-    "pyvista[trame]==0.42.3",
+    "pyvista[jupyter,trame]==0.42.3",
```

As you can see in https://github.com/pyvista/pyvista/pull/5072 . We need to remove `trame` as optional-dependency class, as it is no longer there, and only part of `jupyter` deps.
```suggestion
    "pyvista[jupyter]==0.42.3",
```